Docs/push-credentials-via-conversations-api.md

```diff
@@ -1,6 +1,6 @@
-## Create Push Credentials via the Conversations Credential Resource API
+## Create Push Credentials via the Notify Credential Resource API
 
-Voice SDK users can manage their Push Credentials in the developer console (**Console > Account > Keys & Credentials > Credentials**). Currently the Push Credential management page only supports the default region (US1). To create or update Push Credentials for other regional (i.e. Australia) usage, developers can use the Conversations public API to manage their Push Credentials. Follow the instructions of the [Credential Resource API](https://www.twilio.com/docs/conversations/api/credential-resource) and replace the endpoint with the regional endpoint, for example https://conversations.dublin.ie1.twilio.com for the Australia region.
+Voice SDK users can manage their Push Credentials in the developer console (**Console > Account > Keys & Credentials > Credentials**). Currently the Push Credential management page only supports the default region (US1). To create or update Push Credentials for other regional (i.e. Australia or Ireland) usage, developers can use the Conversations public API against the Notify endpoint to manage their Push Credentials. Follow the instructions of the [Credential Resource API](https://www.twilio.com/docs/conversations/api/credential-resource) and replace the endpoint with the regional endpoint, for example https://notify.sydney.au1.twilio.com for the Australia region or [https://notify.dublin.ie1.twilio.com](https://notify.dublin.ie1.twilio.com) for Ireland region.
```

Comment:
`[https://notify.dublin.ie1.twilio.com](https://notify.dublin.ie1.twilio.com)` can be just `https://notify.dublin.ie1.twilio.com`

Reply:
```suggestion
Voice SDK users can manage their Push Credentials in the developer console (**Console > Account > Keys & Credentials > Credentials**). Currently the Push Credential management page only supports the default region (US1). To create or update Push Credentials for other regional (i.e. Australia or Ireland) usage, developers can use the Credential Resource API to manage their Push Credentials. Follow the instructions of the [Credential Resource API](https://www.twilio.com/docs/conversations/api/credential-resource) and replace the endpoint with the regional endpoint, for example https://notify.sydney.au1.twilio.com for the Australia region or https://notify.dublin.ie1.twilio.com for Ireland region.
```